src/dreammaker/lexer.rs

```diff
@@ -99,6 +100,7 @@ table! {
     "//",  LineComment;
     "/=",  DivAssign;
     ":",   Colon -> CloseColon;
+    ":=",  ExpressAssign;
```

Comment:
Seems like the release notes use the name "assign into" for this

Reply:
Yeah, but other languages (py) use "Assignment Expressions", so I figured that was the more language-agnostic terminology.
https://www.python.org/dev/peps/pep-0572/

Reply:
`:=` being "assignment expression" is a Pythonism relating to the fact that in Python, `=` is a statement, whereas in BYOND `=` is already an expression

Reply:
Hmm. So shall I change this to `IntoAssign`? - seems like the rest are reverse-styled as such.

Reply:
Either word order is good enough, but I like `AssignInto` better. It's not Into/Assign like `+=` is Add/Assign, it's its own thing: Assign Into